src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/VanillaMachinesListener.java

```diff
@@ -97,6 +97,20 @@ public void onAnvil(InventoryClickEvent e) {
         }
     }
 
+    @EventHandler(ignoreCancelled = true)
+    public void onCartographyTable(InventoryClickEvent e) {
+        if (SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_14) && e.getRawSlot() == 2
```

Comment:
You should probably format it like it was done with the Grindstone here:
https://github.com/Slimefun/Slimefun4/pull/2377/files#diff-1d5c31c48a96cc8f0cfae3f72789abd9R43-R47
Because I think a comment explaining why we abort on earlier versions is important to keep there.

Reply:
Okay